src/community/flatgeobuf/README.md

```diff
@@ -0,0 +1,5 @@
+# FlatGeobuf WFS output format prototype
```

Comment:
README works. If you want more people to find and try out this output format, you can add a half a page doc page and enough to package it as a zip for nightly builds. It's going to be somewhat annoying to do, but extends the potential set of testers from people doing code to people that can unzip archives :-D
Anyways, by no means a requirement, and can be done later.